cassandra-3.11/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM openjdk:8u252-slim
 
-ARG STARGATE_VERSION=v1.0.25
+ARG STARGATE_VERSION=1.0.26
```

Comment:
These all need the "v" prefix

Reply:
a-ha... I use the new update script... probably needs fixing :)

Reply:
It was my mistake in specifying the script's arg :facepalm: ... fixed.

Reply:
I wanted to make script accept a tag like value.